app/scripts/models/user.js

```diff
@@ -69,6 +69,14 @@ define(function (require, exports, module) {
       });
     },
 
+    /**
+     * Log the number of stored accounts
+     */
+    logNumStoredAccounts () {
+      const numAccounts = Object.keys(this._accounts()).length;
+      this._metrics.logEventOnce(`num.accounts.${numAccounts}`);
```

Comment:
I went with this format instead of a top level metric because top level metrics are reported on every flush and this minimizes the number of duplicate events.